.storybook/foundations/corner-rounding/corner-rounding.mdx

```diff
@@ -9,3 +11,18 @@ import * as CornerRoundingStories from './corner-rounding.stories.js';
 
 <Story of={CornerRoundingStories.CornerRounding} />
 
+
+## Component side tokens
```

Comment:
Open to suggestions on how we might show or explain this better! I'm going to keep thinking on this one as well.